adage/analyze/tests.py

```diff
@@ -698,6 +699,19 @@ def test_one_edge(self):
         self.call_get_API(uri)
         self.call_non_get_API(uri)
 
+    @staticmethod
+    def get_edge_union(gene_ids):
+        """Return all edges that are related to input gene_ids."""
+        qset = Q(gene1__in=gene_ids) | Q(gene2__in=gene_ids)
+        direct_edges = Edge.objects.filter(qset).distinct()
+        related_genes = set()
+        for e in direct_edges:
+            related_genes.add(e.gene1)
+            related_genes.add(e.gene2)
+        return Edge.objects.filter(
+            gene1__in=related_genes, gene2__in=related_genes
+        ).distinct()
```

Comment:
This follows the same logic implemented above, so if changed above it will also need to change here.